src/routes/views/components/charts/trendChart/trendChart.tsx

```diff
@@ -321,7 +323,7 @@ class TrendChartBase extends React.Component<TrendChartProps, State> {
         gutter={20}
         height={25}
         itemsPerRow={legendItemsPerRow}
-        name="legend"
+        name={chartName ? chartName + '-legend' : 'legend'}
```

Comment:
You can simplify this as

name={`${chartName}-legend`}

It's a required property in the widget trend props.